components/inset-text/src/test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { shallow, mount } from 'enzyme';
+
+import InsetText, { InsetTextNarrowBorder } from './fixtures';
+
+describe('inset text', () => {
+  const wrapperInsetText = mount(<InsetText />);
+  const wrapperInsetTextNarrowBorder = mount(<InsetTextNarrowBorder />);
+
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<InsetText />, div);
+    ReactDOM.render(<InsetTextNarrowBorder />, div);
+  });
+
+  it('should render some inset text', () => {
+    const output = shallow(<InsetText />);
+    expect(output.find('p')).toBeTruthy();
+  });
+
+  it('matches wrapperInsetText snapshot', () => {
+    expect(wrapperInsetText).toMatchSnapshot('wrapperInsetText mount');
+  });
+
+  it('should render some inset text with a narrow border', () => {
```

Comment:
I would adjust the name of this test to be something like `Should render a paragraph element when passed text`. I may be being too particular about the wording of these tests however.